### PR TITLE
DOCSP-42444 updated troubleshooting page with podman error

### PR DESCRIPTION
**Pull request by kyuan-mongodb**
Updated the troubleshooting page to contain a podman error, and updated the v1.26 rn with 2 additional bullets.

Note: I'll backport this to v1.26 after this PR is merged.

- [DOCSP-42444](https://jira.mongodb.org/browse/DOCSP-42444)
- STAGING
  - [Troubleshooting](https://deploy-preview-708--docs-atlas-cli.netlify.app/troubleshooting/#podman-not-found)
  - [v1.26 release notes](https://deploy-preview-708--docs-atlas-cli.netlify.app/atlas-cli-changelog/#atlas-cli-1.26.0)

### Self-Review Checklist

- [ ] Check that the submodule pulled in the right changes (if applicable).
- [ ] [Define](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) taxonomy [values](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) at top of page.
- [ ] Add genre facets (tutorial or reference), as in this [example PR](https://github.com/10gen/cloud-docs/pull/5042).
- [ ] Add programmingLanguage (if necessary).
- [ ] Add meta keywords (if necessary).
- [x] Resolve any new warnings or errors in the build.
- [x] Proofread for spelling and grammatical errors.
- [x] Check staging for rendering issues.
- [x] Confirm links are working.

### CODE REVIEW

File: source/troubleshooting.txt [hunk 1]
```diff
@@ -201,6 +201,25 @@ Alert Config Not Deleted
...
+inside our official docker container, ``mongodb/atlas`` in v1.26 or
+later. Instead, follow the steps decribed in
+:ref:`atlas-cli-deploy-docker`.
+
+You should also inspect past containers and remove any that start with
+``mongod`` or ``mongot`` in your {+cluster+} with the following command:
+
+.. code-block::
+
+   podman ps
```
Comment (fmenezes) [inline source/troubleshooting.txt:217]: there is more to it
```suggestion
   podman ps -a
```

after you inspect containers you can
```
podman container rm -f -v <name or ID>, <name or ID>, ...
```

File: source/troubleshooting.txt [hunk 2]
```diff
@@ -201,6 +201,25 @@ Alert Config Not Deleted
...
 
+podman not found
+~~~~~~~~~~~~~~~~
+
+This error appears if you try to run an ``atlas deployments`` command
+inside our official docker container, ``mongodb/atlas`` in v1.26 or
+later. Instead, follow the steps decribed in
+:ref:`atlas-cli-deploy-docker`.
+
+You should also inspect past containers and remove any that start with
```
Comment (fmenezes) [inline source/troubleshooting.txt:212]: This is optional

```suggestion
You may also inspect past containers and remove any that start with
```

File: source/troubleshooting.txt [hunk 3]
```diff
@@ -201,6 +201,25 @@ Alert Config Not Deleted
...
+``mongod`` or ``mongot`` in your {+cluster+} with the following command:
+
+.. code-block::
+
+   podman ps
+
+.. tip::
+
+   You can safely uninstall podman if you're on MacOS.
```
Comment (fmenezes) [inline source/troubleshooting.txt:221]: if cli was installed via homebrew we can give the command
```
brew uninstall podman
```

Comment (andreaangiolillo) [reply source/troubleshooting.txt:228]: Could we also address this comment?

**Review by andreaangiolillo — APPROVED** [review]
LGTM once the open comment is addressed. Thanks!

File: source/troubleshooting.txt (hunk 3 above)
Comment (kyuan-mongodb) [reply source/troubleshooting.txt:228]: I added it in the last commit. Is there a clearer way to mention this?

Comment (kyuan-mongodb) [reply source/troubleshooting.txt:228]: @andreaangiolillo I added it in the last commit. Is there a clearer way to mention this?

Comment (andreaangiolillo) [reply source/troubleshooting.txt:228]: Sorry I missed it. Thanks!

**Review by andreaangiolillo — APPROVED** [review]

**Pull request merged**